src/field_theory/polynomial_galois_group.lean

```diff
@@ -47,15 +47,45 @@ begin
   rwa [eq_top_iff, ←splitting_field.adjoin_roots, algebra.adjoin_le_iff],
 end
 
-instance [h : fact (p.splits (ring_hom.id F))] : unique p.gal :=
+instance unique_gal_of_splits [h : fact (p.splits (ring_hom.id F))] : unique p.gal :=
```

Comment:
I think there should also be a `lemma` version of this, that doesn't use `fact` and the typeclass system. See also below.